mimicgen/env_interfaces/isaac_lab.py

```diff
@@ -0,0 +1,188 @@
+        Inverse of @target_pose_to_action. Usually used to infer a sequence of target controller poses
+        from a demonstration trajectory using the recorded actions.
+
+        Args:
+            action (np.array): environment action
+            relative (bool): if True, use relative pose actions, else absolute pose actions
+
+        Returns:
+            target_pose (np.array): 4x4 target eef pose that @action corresponds to
+        """
+
+        delta_position = action[:3]
+        delta_rotation = action[3:6]
+
+        # current position and rotation
+        curr_pose = self.get_robot_eef_pose()
+        curr_pos, curr_rot = PoseUtils.unmake_pose(curr_pose)
+
+        # get pose target
+        target_pos = curr_pos + delta_position
+        delta_quat = T.axisangle2quat(delta_rotation)
+        delta_rot_mat = T.quat2mat(delta_quat)
+        target_rot = delta_rot_mat.dot(curr_rot)
+
+        target_pose = PoseUtils.make_pose(target_pos, target_rot)
+        return target_pose
+
+    def action_to_gripper_action(self, action):
+        """
+        Extracts the gripper actuation part of an action (compatible with env.step).
+
+        Args:
+            action (np.array): environment action
+
+        Returns:
+            gripper_action (np.array): subset of environment action for gripper actuation
+        """
+
+        # last dimension is gripper action
+        return action[-1:]
+
+    # isaac_lab-specific helper method for getting object poses
+    def get_object_pose(self, obj_name, obj_type):
+        """
+        Returns 4x4 object pose given the name of the object and the type.
+
+        Args:
+            obj_name (str): name of object
+            obj_type (str): type of object - either "asset", "sensors"
+
+        Returns:
+            obj_pose (np.array): 4x4 object pose
+        """
+        assert obj_type in ["asset", "frame_transformer", "sensor"]
+
+        if obj_type == "asset":
+            # TODO: how do we handle multiple envs in a single scene
+            obj_pos = np.array(self.env.unwrapped.scene[obj_name].root_pos_w()[0]) # Get the pose from the first environment
+            obj_rot = np.array(self.env.unwrapped.scene[obj_name].root_quat_w()[0]) # Get the orienation quaternion from the first environment
+        elif obj_type == "frame_transformer":
+            print(self.env.unwrapped.scene[obj_name].data)
+            obj_pos = self.env.unwrapped.scene[obj_name].data.target_pos_w[0, 0, :].cpu().numpy()
+            obj_rot = self.env.unwrapped.scene[obj_name].data.target_quat_w[0, 0, :].cpu().numpy()
+        elif obj_type == "sensor":
+            raise NotImplementedError(f"Object type '{obj_type}' is not implemented.")
+        else:
+            raise NotImplementedError(f"Object type '{obj_type}' is not implemented.")
+        
+        return PoseUtils.make_pose(obj_pos, PoseUtils.quat2mat(obj_rot))
```

Comment:
Is the `PoseUtils.quat2mat` you added to `PoseUtils` the same as `robosuite.utils.transform_utils.quat2mat`? If true this line can be:
```suggestion
        return PoseUtils.make_pose(obj_pos, T.quat2mat(obj_rot))
```